pkg/stack/stack.go

```diff
@@ -94,6 +97,21 @@ func (s *Stack) GetImageBuilds() (map[string]riov1.ImageBuild, error) {
 				}
 
 				buildConfig[svc.Name] = *svc.Spec.Build
+			} else {
+				// use docker reference parser to determine if its a valid image reference
+				_, err = reference.ParseAnyReference(svc.Spec.Image)
```

Comment:
Any reason we need parse the image? Can we assume that if it begins with `./` then use that as the local reference path?

Reply:
I think path such as `/Users/repo/project/` could also be used? Not guaranteed to be a relative path. But the docker image parsing might be overkill beyond `os.Stat`

Reply:
Looks like this also accepts a commit sha, but that would fail out below. Guessing the benefit in using it is that an invalid image string will fail before the `os.stat` call, even if the file exists.

Reply:
@cbron if its an image SHA than we would essentially do nothing here, which the the correct and current behavior (you are technically referencing an image, not something to build)